pkg/types/ccip/commitstore.go

```diff
@@ -22,6 +22,7 @@ type CommitStoreReader interface {
 	GetExpectedNextSequenceNumber(ctx context.Context) (uint64, error)
 	GetLatestPriceEpochAndRound(ctx context.Context) (uint64, error)
 	IsBlessed(ctx context.Context, root [32]byte) (bool, error)
+	AreBlessed(ctx context.Context, roots [][32]byte) ([]bool, error)
```

Comment:
alphabetical order, please. it's much easier when comparing across multiple files that have different levels of implementation.